modules/flowable-variable-service/src/main/java/org/flowable/variable/service/impl/types/ByteArrayType.java

```diff
@@ -41,6 +52,14 @@ public Object getValue(ValueFields valueFields) {
 
     @Override
     public void setValue(Object value, ValueFields valueFields) {
+        if (value == null) {
+            valueFields.setBytes(null);
+            return;
+        }
+
+        byte[] bytes = (byte[]) value;
+        VariableTypeUtils.validateMaxAllowedLength(maxAllowedLength, bytes.length, valueFields, this);
```

Comment:
@filiphr Is it possible to make variableTypeUtils configurable?
The threshold exception is mandatory in this case.

use case: If we turn on maxAllowedLength the already running process instances (not on the latest definition) can fail.

Reply:
The process instance (v1) is on userTask1 and will create variable decision with value approved-longValue when user decides to approve.
Meantime maxAllowedLength is set to 10, before the approval.
The process instance fails after the approval. Is it possible to replace VariableTypeUtils behavior easily without rewriting all types? (e.g. log a warning instead of exception....)
The same can happen with large variables not only user inputs.

Question: 
Is the length limit applied on the content and indexed entities too? (I know EE).